shell/browser/browser_win.cc

```diff
@@ -603,8 +620,93 @@ v8::Local<v8::Promise> Browser::GetApplicationInfoForProtocol(
   return handle;
 }
 
-bool Browser::SetBadgeCount(int count) {
-  return false;
+bool Browser::SetBadgeCount(base::Optional<int> count) {
+  if (count.has_value() && count.value() == 0) {
+    badge_content_ = base::nullopt;
+  } else {
+    badge_content_ = badging::BadgeManager::GetBadgeString(count);
+  }
+  badge_process_id = GetCurrentProcessId();
+  EnumWindows(WindowsEnumerationBadgeHandler,
```

Comment:
@zcbenz is there a path to get a HWND from electron::WindowList?  I followed what was done here https://github.com/electron/electron/blob/master/shell/browser/browser_win.cc#L46 because I needed a HWND.

Reply:
You can get the HWND via `GetAcceleratedWidget()` method:

```c++
for (auto* window : WindowList::GetWindows()) {
  HWND hwnd = window->GetAcceleratedWidget();
}
```

I'll do a rewrite of `Browser::Focus` implementation, it is rather outdated code.